src/test/java/seedu/address/model/util/IncomeComparisonOperatorTest.java

```diff
@@ -0,0 +1,89 @@
+package seedu.address.model.util;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class IncomeComparisonOperatorTest {
```

Comment:
Very robust tests! You might want to add a test case for the `trim()` functionality in `isValidComparisonOperator ` method to double check too.